manage/knowledge-base.rst

```diff
@@ -197,3 +197,32 @@ Delete
 Use this panel to delete the knowledge base.
 If you wish to unpublish it without deleting all of its content,
 simply disable it via the toggle button at the top of the window instead.
+
+Permissions
+-----------
+
+The general access to the knowledge base is managed via
+``knowledge_base.reader`` and ``knowledge_base.editor`` permissions. You can
+find more information about roles and permissions
+:doc:`here <roles/index>`.
+
+If you want to grant more granular permissions than to global read/edit
```

Comment:
Looks like there's a mistype in „permissions than to global read/edit“ . I guess it was supposed to be „permissions than global read/edit“?